core/src/main/scala/kafka/controller/ControllerEventManager.scala

```diff
@@ -140,6 +143,9 @@ class ControllerEventManager(controllerId: Int,
     }
   }
 
+  // for testing
+  private[controller] def setControllerEventThread(thread: ControllerEventThread): Unit = this.thread = thread
```

Comment:
We can remove this since we have `private[controller] var thread` with the same visibility.